.github/workflows/ci.yml

```diff
@@ -18,11 +18,10 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
-          - '3.1'
```

Comment:
Ruby 3.1 EOL is 2025-03-31
See: https://www.ruby-lang.org/en/downloads/branches/